src/tls13.c

```diff
@@ -7006,6 +7345,33 @@ int DoTls13Finished(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
     }
 #endif
 
+#ifdef WOLFSSL_DTLS13
+        if (ssl->options.dtls) {
+
+            WOLFSSL_DTLS_PEERSEQ* peerSeq = ssl->keys.peerSeq;
+
+#ifdef WOLFSSL_MULTICAST
+            if (ssl->options.haveMcast) {
+                peerSeq += ssl->keys.curPeerId;
+                peerSeq->highwaterMark = UpdateHighwaterMark(0,
+                                                             ssl->ctx->mcastFirstSeq,
+                                                             ssl->ctx->mcastSecondSeq,
+                                                             ssl->ctx->mcastMaxSeq);
+            }
+#endif
+
+            peerSeq->nextEpoch++;
```

Comment:
This should be incremented by 2 (unless you have already implemented early data support).